PowerPointLabs/PowerPointLabs/PositionsLab/PositionsPaneWPF.xaml.cs

```diff
@@ -1062,7 +1062,7 @@ public void ExecutePositionsAction(Action<ShapeRange> positionsAction, bool isPr
                     this.StartNewUndoEntry();
                 }
 
-                simulatedShapes = DuplicateShapes(selectedShapes);
+                simulatedShapes = DuplicateShapes(selectedShapes); // selectedShapes.Duplicate() may return a list with revresed sequence  
```

Comment:
Typo in the comments

Reply:
do shift your comments to the line above. 
read our styleguide for more info.